src/ui/components/ReplayAssistButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, useEffect, useState } from "react";
+
+import useLocalStorage from "replay-next/src/hooks/useLocalStorage";
+import { UIThunkAction } from "ui/actions";
+import { setSelectedPrimaryPanel } from "ui/actions/layout";
+import { getReplayAssist, replayAssistToggled } from "ui/reducers/app";
+import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
+import useAuth0 from "ui/utils/useAuth0";
+
+function toggleAndShowAssistPanel(): UIThunkAction {
+  return (dispatch, getState) => {
+    dispatch(replayAssistToggled());
+    const newValue = getReplayAssist(getState());
+    if (!newValue) {
+      dispatch(setSelectedPrimaryPanel("events"));
```

Comment:
Toggling one panel off should just hide that panel. It shouldn't show another panel. That's how the other panels work:

https://user-images.githubusercontent.com/29597/229527858-b0874d2f-424c-4ba9-bb3e-90d380005cdf.mp4

Reply:
In this case it's coming from the toggle itself, not a menu element ->

![MenuToggle](https://user-images.githubusercontent.com/9154902/230036701-5e46aa9c-56ec-409d-b61f-0a8967268b9e.gif)

Without this text, you can toggle off the Replay Assist pane, but then nothing is selected on the side, leading to a broken experience.